### PR TITLE
ROX-18669: Add listening endpoints info

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):
4.2+

[Issue](https://issues.redhat.com/browse/ROX-18669)

[Link to docs preview
](https://file.rdu.redhat.com/kcarmich/ROX-18669-listening-ports/operating/audit-listening-endpoints.html)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by JoukoVirtanen — APPROVED** [review]
LGTM!

File: operating/audit-listening-endpoints.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+You can further filter the information displayed on the page by using the filter field and entering individual deployments, namespaces, and clusters.
+
+//"expand icon" is used elsewhere in docs
+Click the expand icon at the top of the list to expand all sections for all deployments listed, or click the expand icon on a single deployment line to view additional information about that deployment. The following information is provided:
+
+* Exec file path: Location of the process
+* PID: System ID of the process
+* Port: Port on which the process is listening
+* Protocol: Protocol in use by the process
+* Pod ID: Name of the pod in which the process is contained
```
Comment (tmalove) [inline operating/audit-listening-endpoints.adoc:33]:
```suggestion
* Pod ID: Name of the pod where the process is contained
```
Just a suggestion for minimalism to consider.

File: _topic_map.yml [hunk 2]
```diff
@@ -197,6 +197,8 @@ Topics:
   File: default-security-policies
 - Name: Managing network policies
   File: manage-network-policies
+- Name: Auditing listening endpoints
```
Comment (tmalove) [inline _topic_map.yml:200]: I was going to leave a reference for changing the title of the concept module to `Audit listening endpoints`, but I see in the navigation that the adjacent concept sections (Managing, Reviewing...) follow the same style.

**Review by tmalove — APPROVED** [review]

**Comment by kcarmichael08** [comment]
Thank you @tmalove !

**Pull request merged**